pkg/reconciler/service/install.go

```diff
@@ -89,6 +89,10 @@ func (r *Install) renderManifest(chartProvider chart.Provider, model *reconciler
 	if err != nil {
 		msg := fmt.Sprintf("Failed to get manifest for component '%s' in Kyma version '%s'",
 			model.Component, model.Version)
+		if model.URL != "" {
```

Comment:
I think we should also enhance the log-messagse in `git/clone.go` when it'S complaninig about the not found revision:  https://github.com/kyma-incubator/reconciler/blob/main/pkg/reconciler/git/clone.go#L79 (hope hte URL is accessbiel over the `repo` struct).

Reply:
you're right. I added this msg in clone.go